### PR TITLE
Import config from file gui

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
## Description of changes
Added an button on the gui config panel to load config settings from file.
The core functionality already exists so this just brings it into the gui

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

**Comment by ericenns** [comment]
I attempted to import a bad config file and the application immediately crashed. See bad_irida_conf.conf in attached zip.

[bad_irida_conf.zip](https://github.com/phac-nml/irida-uploader/files/8215925/bad_irida_conf.zip)

File: iridauploader/gui/config.py [hunk 1]
```diff
@@ -68,13 +70,20 @@ def __init__(self, parent=None):
         self._get_settings_from_file()
         self._contact_irida()
 
+        # Error message popup
+        self._error_message_box = QtWidgets.QMessageBox(parent=self)
+        self._error_message_box.setText("Config file was invalid. Please check your file and try again.")
```
Comment (ericenns) [inline iridauploader/gui/config.py:75]: Do we have a read the docs for the irida-uploader with a page on the config file showing the format that we could link to?

**Review by ericenns — CHANGES_REQUESTED** [review]
No longer crashes on bad file which is great. I just have one comment that I left below.

File: iridauploader/gui/config.py (hunk 1 above)
Comment (JeffreyThiessen) [reply iridauploader/gui/config.py]: Added.

**Pull request merged**